shell/platform/windows/windows_registry.h

```diff
@@ -0,0 +1,33 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_WINDOWS_WINDOWS_REGISTRY_H_
+#define FLUTTER_SHELL_PLATFORM_WINDOWS_WINDOWS_REGISTRY_H_
+
+#include <Windows.h>
+
+#include "flutter/fml/macros.h"
+
+namespace flutter {
+
+class WindowsRegistry {
+ public:
+  explicit WindowsRegistry();
```

Comment:
`explicit` isn't needed here since there's no parameter to be eligible for implicit conversion.

```suggestion
  WindowsRegistry() = default;
```